File: computer3/Fall Term/Solutions/Class06_Oct_16/lvl3_just_stalling.py
import sys
input = sys.stdin.readline
n = int(input())
cow = list(map(int, input().split()))
stall = list(map(int, input().split()))
cow.sort(reverse=True)
stall.sort(reverse=True)
ans = 1
for i in range(n):
    cnt = 0
    for j in range(i, n):
        if stall[j] >= cow[i]:
            cnt += 1
    ans = ans*cnt
print(ans)

File: computer3/Fall Term/Solutions/Class06_Oct_16/lvl3_just_stalling.cpp
#include <bits/stdc++.h>
using namespace std;
int n; long long ans=1;
int main(){
    cin >> n;
    vector<int> cow(n), stall(n);
    for(int i=0; i<n; i++) {
        cin >> cow[i];
    }
    for(int i=0; i<n; i++) {
        cin >> stall[i];
    }
    sort(cow.begin(), cow.end());
    sort(stall.begin(), stall.end());
    for(int i=n-1; i>=0; i--){
        int cnt = 0;
        for(int j=i; j>=0; j--) {
            if(stall[j] >= cow[i]) cnt++;
        }
        ans *= cnt;
    }
    cout << ans << endl;
}